lib/services/ios-project-service.ts

```diff
@@ -400,6 +437,23 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		}).future<void>()();
 	}
 
+	private validateStaticLibrary(libraryPath: string): IFuture<void> {
+		return (() => {
+			if (path.extname(libraryPath) !== ".a") {
+				this.$errors.failWithoutHelp("The bundle at %s does not contain valid '.a' extension.", libraryPath);
```

Comment:
This message is weird. Can we have an invalid ".a" extension? Ping @ikoevska for help.